### ISSUE TITLE
All I get is "it works!" for my new site in virtualhost 7

### ISSUE THREAD

**Issue by retsoced** [clickontyler/VirtualHostX-Issues#53]
I just upgraded to version 7 so a lot of the vagrant stuff is not as familiar to me, but the vm doesn't seem to be using the files from the webroot location to serve files since I just get the default "it works!" page. I need to have PHP enables and mod rewrite which seem to be both installed and available.

the sites-enabled and sites-availble files are both empty so I'm not sure what's going on and why it isn't sharing the web files properly.

**Comment by tangst** [comment]
I have the same problem, and I don't have a dynamic site.

**Comment by tangst** [comment]
I somehow got this to work.  I ended up having to destroy the domain and virtual machine and recreate them.  Then, I had to change the domain name to something else and then change the domain back.  That was a weird hiccup.

**Comment by tangst** [comment]
Spoke too soon.  The error is back.  It seems VHX7 loses the folder mapping after rebooting my Mac.

**Comment by burblebeep** [comment]
This started happening to me too after upgrading to OS 10.12.3. Might be a clue in the VHX startup:

=> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default: 
    default: Guest Additions Version: 4.3.36
    default: VirtualBox Version: 5.0

**Comment by tangst** [comment]
I will have to shut down and start up VHX to see what is displaying in the VHX terminal window.  Unfortunately, I am not a VM expert, so I have no idea how to upgrade Guest Additions.  This sounds like I have to go to VirtualBox itself, but I'm concerned that if I muck the VirtualBox settings, VHX7 gets messed up in the process.

**Comment by burblebeep** [comment]
I destroyed and rebuilt my VM and I'm kind of up and running again. I have multiple hosts available but instead of using the short Domain Name like I've used previously ```http://mydomain/```, I needed to enter the full domain in my browser location ```http://mydomain.vhx.host:8080/``` also my .htaccess file seems to be ignored as well as the full paths used by PHP have changed. I decided to try running 'do-release-upgrade' to just see if I can break it again.

**Comment by tangst** [comment]
I have no idea how to update the Guest Additions on the Ubuntu box.  I don't have any experience with Ubuntu or Vagrant.

It seems that I get best results if I close VirtualHostX to allow it to shut down the VM.  If I just shut down the Mac, it seems it doesn't give enough time for VHX to shut down the VM, which might cause  the mapping to get all screwed up.

**Comment by burblebeep** [comment]
Don't know what to say. I haven't had much luck either and whatever I had working yesterday isn't working today. Any updates I tried to make to the VM didn't help. Back to "It works!"

Here's what seemed to work yesterday:

Virtual Machine > Destroy
[wait for VM to shutdown]
VHX Setup
Step 3. Download and setup the web server
[wait for VM to rebuild]
VHX Menu
Website > Apply Changes CMD-S
[wait for VM to restart with correct Domain Names]

I also ran Virtual Machine > Bind to Ports 80 and 443 so I didn't have to add port numbers to the domain names. the only thing that didn't work at all was my custom .htaccess rules.  But now nothing works locally and I haven't heard back from support.
